.github/workflows/code-quality.yml

```diff
@@ -32,20 +32,6 @@ jobs:
         run: mvn -B -T1C -DskipTests -DskipChecks install
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v2
-  spotbug:
-    name: SpotBugs
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3.3.0
-        with:
-          distribution: 'temurin'
-          java-version: '17'
-          cache: 'maven'
-      - name: SpotBugs
-        run: mvn -B -T1C -DskipTests -DskipChecks verify -Pspotbugs -Dspotbugs.skip=false
```

Comment:
:question: There is this `-Dspotbugs.skip=false` which is not part of the changed code. Are you positive that this is not needed?

Reply:
Based on the logs here: https://github.com/camunda/zeebe/runs/6622785922?check_suite_focus=true it looks like it does run. The default value is also `false`: https://spotbugs.github.io/spotbugs-maven-plugin/check-mojo.html#skip